### PR TITLE
go: add support for gometalinter

### PR DESCRIPTION
**Pull request by buztard**
This adds support for https://github.com/alecthomas/gometalinter
if available

### CODE REVIEW

File: autoload/neomake/makers/ft/go.vim [hunk 1]
```diff
@@ -1,7 +1,15 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#go#EnabledMakers()
-    return ['go', 'golint']
```
Comment (benekastah) [inline autoload/neomake/makers/ft/go.vim]: You can just add 'gometalinter' to the end of this list. Neomake checks that a maker exists before it tries to run it.

File: autoload/neomake/makers/ft/go.vim [hunk 2]
```diff
@@ -27,3 +35,15 @@ function! neomake#makers#ft#go#golint()
             \ '%-G%.%#'
         \ }
 endfunction
+
+function! neomake#makers#ft#go#gometalinter()
+    return {
+        \ 'args': ['-t', '%:p:h'],
```
Comment (blampe) [inline autoload/neomake/makers/ft/go.vim]: Consider passing -j here. The default setting of 16 concurrency can slam a system on larger projects.

File: autoload/neomake/makers/ft/go.vim (hunk 1 above)
Comment (blueyed) [inline autoload/neomake/makers/ft/go.vim]: `gometalinter` sounds like it should be preferred over other linters (from the name of it) - similar to `pylama` on Python.

**Comment by blueyed** [comment]
@buztard 
Is this still a thing?
Please update the PR according to the feedback and rebase it.

**Comment by blueyed** [comment]
Closing this for now.
Please re-open or pick it up into a new PR.

**Pull request closed without merging**